src/lib/utils/application-commands/ApplicationCommandRegistry.ts

```diff
@@ -32,6 +32,9 @@ export class ApplicationCommandRegistry {
 	public readonly contextMenuCommands = new Set<string>();
 	public readonly guildIdsToFetch = new Set<string>();
 
+	public globalCommandId: string | null = null;
+	public readonly guildCommandIds = new Map<string, string>();
```

Comment:
This would suggest the Map would have to be `Map<string, string | null>`

Reply:
Why?

Reply:
Just seems weird that one is a single constant and the other is a map to hold multiple?

Reply:
if `globalCommandId` is nullable then wouldn't the id in the Map also be nullable?

Reply:
Nop, as we set a command id for a guild command only when registered
The global one has nullable id because... I just chose it over undefined